connect/src/com/telenor/connect/ConnectRefreshTokenMissingException.java

```diff
@@ -0,0 +1,19 @@
+package com.telenor.connect;
+
+public class ConnectRefreshTokenMissingException extends IllegalStateException {
```

Comment:
Are we not using checked exceptions? Is this an unrecoverable condition? Is this case really an illegal state?